### PR TITLE
[Certificates] Background image abstraction

### PR DESCRIPTION
**Pull request by legionth**

### CODE REVIEW

File: Services/Certificate/classes/Form/Repository/class.ilCertificateSettingsFormRepository.php [hunk 1]
```diff
@@ -348,4 +357,49 @@ public function fetchFormFieldData(string $content)
...
         return $this->formFieldParser->fetchDefaultFormFields($content);
     }
+
+    /**
+     * Checks for the background image of the certificate
+     *
+     * @return boolean Returns TRUE if the certificate has a background image, FALSE otherwise
+     * @throws ilException
+     */
+    private function hasBackgroundImage()
```
Comment (mjansenDatabay) [inline Services/Certificate/classes/Form/Repository/class.ilCertificateSettingsFormRepository.php]: This should not be located in a controller.

File: Services/Certificate/classes/Placeholder/Values/class.ilDefaultPlaceholderValues.php [hunk 2]
```diff
@@ -134,7 +134,14 @@ public function getPlaceholderValues(int $userId, int $objId) : array
         $placeholder['USER_LASTNAME']      = $this->utilHelper->prepareFormOutput((trim($user->getLastname())));
         $placeholder['USER_TITLE']         = $this->utilHelper->prepareFormOutput((trim($user->getUTitle())));
         $placeholder['USER_SALUTATION']    = $this->utilHelper->prepareFormOutput($this->language->txt("salutation_" . trim($user->getGender())));
-        $placeholder['USER_BIRTHDAY']      = $this->utilHelper->prepareFormOutput((trim($user->getBirthday())));
+
+        $birthday = '';
```
Comment (mjansenDatabay) [inline Services/Certificate/classes/Placeholder/Values/class.ilDefaultPlaceholderValues.php]: @legionth Does this really belong to this PR?

**Review by mjansenDatabay — CHANGES_REQUESTED** [review]
Thx @legionth , only one issue left.

File: Services/Certificate/classes/Placeholder/Values/class.ilDefaultPlaceholderValues.php (hunk 2 above)
Comment (legionth) [reply Services/Certificate/classes/Placeholder/Values/class.ilDefaultPlaceholderValues.php]: Hm this should already have been in trunk, I will check this

Comment (mjansenDatabay) [reply Services/Certificate/classes/Placeholder/Values/class.ilDefaultPlaceholderValues.php]: Thx

Comment (legionth) [reply Services/Certificate/classes/Placeholder/Values/class.ilDefaultPlaceholderValues.php]: Fixed via rebased. Ready for Review!

**Review by mjansenDatabay — APPROVED** [review]

**Pull request merged**